docs/core/deploying/single-file.md

```diff
@@ -139,18 +139,26 @@ These properties have the following functions:
 Publish a single file application using the [dotnet publish](../tools/dotnet-publish.md) command. When you publish your app, set the following properties:
 
 - Publish for a specific runtime: `-r win-x64`
```

Comment:
It's a little odd that we are mixing `-r` (the command line option) here and the project setting for `PublishSingleFile` in the next line. I think it would make sense to be consistent here.